functions/UpdateBookingSlot-BookToISV/run.csx

```diff
@@ -1,4 +1,8 @@
 ﻿#load "..\Shared\httpUtils.csx"
+#load "..\Shared\CompleteAppointmentDTO.csx"
+#load "..\Shared\TechnicalResource.csx"
+#load "..\Shared\EncryptionUtils.csx"
+#load "..\Shared\isv.csx"
```

Comment:
I'm not convinced this will restore properly in the actual functions environment due to how nuget works with shared #load code. In the new 2017-style compilation model this will work. 

Also, I explicitly moved away from shared data dependencies as these are supposed to be nano-services. 

If you have found a way to make this work (the current workaround is to compile the shared classes into a dll and import it statically by including the .dll file in the bin folder, which I don't really want to do here) then we should do the same for all of the other nano-services in the function project so they are consistent and we only have one definition of ISV.